src/pages/using-spark/components/award.mdx

```diff
@@ -20,18 +19,14 @@ showcasing Awards with a disclaimer.
 
 ### Usage
 
-Use Award to showcase company accolades
-with a disclaimer. Award is a
-standard component of the
-[Footer](/using-spark/components/footer).
+Display company accolades and disclaimers on a page or in a [Footer](/using-spark/components/footer).
```

Comment:
Seems arbitrary when we link to other components or not. We should either always do it or never.

Reply:
I think they are good here, it gives the user a reference to what they need. I think we should always link to the component when it's relevant.